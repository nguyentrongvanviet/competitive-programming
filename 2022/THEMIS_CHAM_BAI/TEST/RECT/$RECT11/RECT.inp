36
254356483 -58029162
-434416629 -311795877
-11714081 359978059
896671291 990338946
144437045 204388106
-870224648 668753963
263322378 -20651004
-28342050 725474159
67249630 679392776
-325172555 823108968
-420075368 50588112
934214002 -613965953
-944372389 739550797
-569100744 -407183298
-223924682 345351678
-453696839 536579209
-420075368 50588112
-275175006 39747385
-216896981 -663078204
337154050 742102128
-783644248 -500026294
-489242358 748016885
543125496 -171647581
934214002 -613965953
179603677 24678728
696343486 377751837
353782019 376606028
-95684738 210733977
489548942 -338985116
984584060 85132125
964489766 -554242436
-489242358 748016885
-490458802 227414629
489548942 -338985116
984584060 85132125
-403414950 174365086
