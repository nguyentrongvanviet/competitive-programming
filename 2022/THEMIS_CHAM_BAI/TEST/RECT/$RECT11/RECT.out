133863888468274961